src/classes/UTIL_RecordTypes.cls

```diff
@@ -107,11 +107,7 @@ public class UTIL_RecordTypes {
         Map<Id, Schema.RecordTypeInfo> rtiById = objectType.getDescribe().getRecordTypeInfosById();
         if (rtiById.containsKey(recordTypeId)) {
             Schema.RecordTypeInfo rti = rtiById.get(recordTypeId);
-            if (!rti.isMaster() && rti.isAvailable()) {
```

Comment:
so it looks like you would not return a name if it was the Master record type.  Now you will return it.  Is that a problem?